src/smart-components/role/add-role-new/add-permissions.js

```diff
@@ -32,23 +32,42 @@ export const accessWrapper = (rawData, filters = { applications: [], resources:
     );
 
     return {
-        data,
+        data: data.filter(permission => filterSplats(permission)),
         filteredData,
         applications: [ ...new Set(data.filter(permission => filterResource(permission) && filterOperation(permission)).map(({ application }) => application)) ],
         resources: [ ...new Set(data.filter(permission => filterApplication(permission) && filterOperation(permission)).map(({ resource }) => resource)) ],
         operations: [ ...new Set(data.filter(permission => filterApplication(permission) && filterResource(permission)).map(({ operation }) => operation)) ]
     };
 };
 
+export const resolveSplats = (selectedPermissions, permissions) => {
+    return (permissions.length > 0
+        ? selectedPermissions.map(({ uuid: permission }) => {
```

Comment:
I wouldn't use `map` and `flat` as some older browsers are not supporting it fully (hence the new depedency for core-js right?), I'd go with `flatMap` from lodash in this example `import flatMap from 'loadash/flatMap';`